### DISCUSSION TITLE
Is lance quicker than parquet on local disk workload?

### DISCUSSION THREAD

**Discussion by JackTan25** [lance-format/lance#1612] [Q&A]
I'm interested in lance recently, but as you said, it's design for cloud, but My application is working on local disk, we have no need to upgrade to cloud storage. So for my case, if I use lance, can I get performance improvement?

**Comment by eddyxu** [comment]
@JackTan25 Lance works pretty well on SSDs , especially NVMe SSD. It favors the storage media, which has consistently low random i/o latency.

**Reply by JackTan25** [reply]
ok, thanks. How does lance do optimizations for vector store and accelerate the vector search?
